tensorboard/plugins/npmi/README.md

```diff
@@ -17,20 +17,23 @@ To make use of the plugin, the following steps are required.
 - Mandatory: Exported nPMI data, annotations, and metrics. Metrics for npmi data need to be prefixed with `nPMI@` or `nPMI_diff@`. To export your nPMI calculations for use with this plugin, use our summary generators in `summary.py`.
 - Optional: In addition to nPMI calculations, you can also export count values for annotation/metric pairs. This can be used as an indicator for how reliable the nPMI values are. Count values are provided as additional metrics prefixed with `count@`, and need to be provided to the summary exporters together with the nPMI values.
 
-(If you want to experiment with the plugin but do not have your own data, take a look at demo data described below)
+#### Demo Data
+
+If you want to experiment with the plugin but do not have your own data, simply generate demo data using `bazel run //tensorboard/plugins/npmi:npmi_demo`.
+
+#### Converting CSV Data
+
+If you have csv data containing annotations as rows, and metrics in the first row of the csv, you can convert this data to logfiles which can be displayed by the plugin. To do this, run `bazel run //tensorboard/plugins/npmi:csv_to_plugin_data -- --csv_path [path_to_your_csv_file]`.
```

Comment:
Let's treat the script for now as an example script, which provides a reference implementation for demonstration purposes.

Can we
- Rename the Bazel target to `csv_to_plugin_data_demo`
- Update the 2nd sentence in this section to something like: "As an example, you can run this demo target `bazel run //tensorboard/...csv_to_plugin_data_demo..."